octopusdeploy/schema_action_apply_terraform_template.go

```diff
@@ -107,6 +107,46 @@ func addTerraformTemplateAzureAccountSchema(element *schema.Resource) {
 	}
 }
 
+func addTerraformTemplateGoogleAccountSchema(element *schema.Resource) {
+	element.Schema["google_account"] = &schema.Schema{
```

Comment:
I would recommend we change the name to `google_cloud_account`.

Reply:
Done